input/otlp/traces.go

```diff
@@ -411,8 +410,17 @@ func TranslateTransaction(
 
 			// messaging.*
 			case "message_bus.destination", semconv.AttributeMessagingDestination:
-				message.QueueName = stringval
+				if event.Transaction.Message == nil {
+					event.Transaction.Message = modelpb.MessageFromVTPool()
```

Comment:
[For reviewer] Note: IMO there are still quite a few cases where we are doing allocs in this function, when we should use vt pool instead. This change fixes one.

Reply:
I saw the improvement - it would make sense to open a tech debt issue, collecting places where allocations could be reduced.